exercises/affine-cipher/src/test/java/AffineCipherTest.java

```diff
@@ -0,0 +1,136 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public class AffineCipherTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testEncodeYes() {
+        AffineCipher affineCipher = new AffineCipher();
+
+        assertEquals("xbt", affineCipher.encode("yes", 5, 7)
+        );
+    }
+
+    @Ignore("Remove to run test.")
```

Comment:
Not very important, but to be consistent with the other exercises there shouldn't be a full stop after "test" 🙂